samples/MemoryCacheFileWatchSample/project.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0",
+      "type": "platform"
+    },
+    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.0.0",
```

Comment:
Way too many dependencies for the caching repo. We can't have any web-apps here, only console apps.